src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -160,6 +157,10 @@ private async Task ProcessRequestCoreAsync(StreamingMessage request)
                     Result = new StatusResult { Status = StatusResult.Types.Status.Success }
                 };
             }
+            else if (request.ContentCase == MsgType.InvocationCancel)
```

Comment:
Potential switch statement refactoring? (feel free to ignore if you don't want to).

Reply:
We made an [issue](https://github.com/Azure/azure-functions-dotnet-worker/issues/930) for this but I can just address is now since it keeps coming up :)

Reply:
Was not aware of the separate issue. Someone is working on it in a PR already, probably best to leave it to that other PR then.

Reply:
It looks like that PR closed and was doing a major refactor so I think it might be better to just resolve that here